app/src/main/java/org/mozilla/webmaker/activity/Play.java

```diff
@@ -6,13 +6,14 @@
 import android.os.Bundle;
 import android.view.Menu;
 import android.view.MenuItem;
+
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.mozilla.webmaker.R;
 import org.mozilla.webmaker.WebmakerActivity;
-import org.mozilla.webmaker.web.javascript.WebAppInterface;
 import org.mozilla.webmaker.router.Router;
 import org.mozilla.webmaker.util.Share;
+import org.mozilla.webmaker.web.javascript.WebAppInterface;
```

Comment:
again probably better to just revert these changes, since they aren't relevant to the patch itself, and don't actually change anything in how things are compiled.